keel-sqs/src/main/kotlin/com/netflix/spinnaker/config/SqsConfiguration.kt

```diff
@@ -53,5 +53,7 @@ class SqsConfiguration {
     objectMapper: ObjectMapper,
     actuator: ResourceActuator
   ): Any =
-    SqsResourceCheckListener(sqsClient, queueARN, objectMapper, actuator)
+    sqsClient.createQueue(queueARN.name).run {
```

Comment:
Feel like there might be a better place to do the queue creation than here but probably ok for now.